packages/core/src/service/fetcher.ts

```diff
@@ -15,7 +15,9 @@ export async function fetchWithCache(
 		request.headers.set('If-None-Match', cachedEtag)
 	}
 	try {
-		const response = await web.fetch(request)
+		const response = await web.fetch(request, {
+			headers: { 'User-Agent': 'SpyglassMC (+https://spyglassmc.com)' },
```

Comment:
Perhaps we could include the vscode extension version number here? Or if that is too impractical, maybe the core package version?

Reply:
I wonder how to get that number at runtime though, since depending on how the package is used/bundled the location of `package.json` (if it even exists) could anywhere. I guess we could store it as a constant somewhere and modify it in the `release` script?